spec/jobs/rapid_ready_for_decision/form526_asthma_job_spec.rb

```diff
@@ -54,4 +41,14 @@
       end
     end
   end
+
+  describe '#assess_data', :vcr do
+    subject { RapidReadyForDecision::Form526AsthmaJob.new.assess_data(submission) }
```

Comment:
This will facilitate class renaming (which is part of my PR):
```suggestion
    subject { described_class.new.assess_data(submission) }
```